vnext/Microsoft.ReactNative/IReactContext.idl

```diff
@@ -2,13 +2,20 @@
 // Licensed under the MIT License.
 
 import "IJSValueWriter.idl";
+import "ReactPropertyBag.idl";
 
 #include "NamespaceRedirect.h"
 
 namespace Microsoft.ReactNative {
 
   [webhosthidden]
   interface IReactContext {
+    // A singleton property bag global for the whole application.
+    ReactPropertyBag GlobalProperties { get; };
+
+    // Properties associated with the ReactNativeHost and instances it creates.
```

Comment:
>ReactNativeHost and instances [](start = 38, length = 29)

For my own education: Can a ReactNativeHost be associated with multiple instances?

Reply:
No

Reply:
One RNH -> 0..1 RN instances